src/libraries/System.Data.Common/src/System/Data/SQLTypes/SQLDecimal.cs

```diff
@@ -2567,13 +2570,13 @@ private static uint LO(ulong x)
         //
         private static void MpDiv
         (
-        uint[] rgulU,      // In    | U
+        ReadOnlySpan<uint> rgulU,      // In    | U
         int ciulU,      // In    | # of digits in U
-        uint[] rgulD,      // In    | D
+        Span<uint> rgulD,      // In    | D // this is written to, should is also be InOut ?
```

Comment:
I assume this comment change is temporary?  We shouldn't be adding questions in comments like this :)

Reply:
Yes temporary until someone with a deeper understanding of the algorithm being implemented looks and indicates that it's either OK or directs a change. This is why I called it out in the PR body text. It might be fine but I'd like signoff on that.

Reply:
Would master of real numbers @tannergooding be the right person to ask on this?

Reply:
Not sure it's Tanner, but cc @tannergooding

Reply:
I don't believe you typically mark arrays written to as `out` (it wasn't marked as such when it was an array). However, I also don't know the exact convention being followed here.

Reply:
We don't use `[InOut]` or similar attributes except when dealing with p/invoke.

Reply:
Oh, I think you're just talking about the code comment, not an attribute. Sure, mark the comment as _InOut_ to match what everything else has.

Reply:
Done.